openshift/README.md

```diff
@@ -86,16 +82,62 @@ helm repo add patroni-chart https://bcgov.github.io/nr-patroni-chart
 helm install -n <namespace> met-patroni patroni-chart/patroni
 ```
 
-If HA is not necessary create two instances of a postgresql database (transactional and analytics):
+If HA is not necessary create a instance of a postgresql database:
 
 ```
-oc new-app --template=postgresql-persistent -p POSTGRESQL_DATABASE=met -p DATABASE_SERVICE_NAME=postgresql-met
+oc new-app --template=postgresql-persistent -p POSTGRESQL_DATABASE=app -p DATABASE_SERVICE_NAME=met-postgresql
 ```
 
-```
-oc new-app --template=postgresql-persistent -p POSTGRESQL_DATABASE=met-analytics -p DATABASE_SERVICE_NAME=postgresql-edw
-```
+### Setup
+
+1. Users Setup script is located at ./postgresql-user-setup.sql
+1. Initial database setup script is located at ./postgresql-schema-setup.sql
+1. Openshift secret yaml is located at ./database-users.secret.yml
+
+## Restore Backup Script
+
+Backups are generated daily by the dc "backup" in the test and production realms and are composed by a SQL script containing the database structure + data.
+
+To restore the backup follow these steps:
+
+1. Connect to openshift using the terminal/bash and set the project (test/prod).
+1. Transfer the backup file to your local using the command below:
+
+    ```bash
+    oc rsync <backup-pod-name>:/backups/daily/<date> <local-folder>
+    ```
+
+    This copies the folder and contents from the pod to the local folder.
+
+1. Extract backup script using gzip:
```

Comment:
all 1. ?

Reply:
its a mark down thing, it will be interpreted as a numeric list and it automatically handles the number by order